charts/celery-exporter/Chart.yaml

```diff
@@ -12,6 +12,4 @@ sources:
 maintainers:
   - name: danihodovic
   - name: adinhodovic
-
-version: 0.7.0
-appVersion: 0.9.2
```

Comment:
Did we intend to remove this line?

Reply:
Yes, in preference to dynamically setting it as done [below with the `--app-version` flag](https://github.com/danihodovic/celery-exporter/blob/cf5944ac30c20ffdc2739919c4be3658b19a83cc/.github/workflows/release.yml#L97) ...

https://github.com/danihodovic/celery-exporter/blob/cf5944ac30c20ffdc2739919c4be3658b19a83cc/.github/workflows/release.yml#L97

As it should track the app version which is only determined during the actual release.